### DISCUSSION TITLE
Updated Block Limits

### DISCUSSION THREAD

**Discussion by AshtonStephens** [stacks-network/stacks-core#4198] [Blockchain]
# Updated Block Limits

![Required For SIP](https://img.shields.io/badge/Required_for_SIP-yes-red) ![Priority Medium](https://img.shields.io/badge/priority-medium-yellow)

After we are able to benchmark Nakamoto blocks we'll need to determine updated block limitations. This cannot be done effectively until much of the functionality has been implemented.

> Details to follow…
> We'll link any related tickets here below:
> ...

**Comment by AcrossfireX** [comment]
Here is a relevant comment raised on discord about this: https://discord.com/channels/621759717756370964/730814571517968436/1172584525113606214
